app/models/rummager_search.rb

```diff
@@ -22,10 +23,19 @@ def rummager_search
   end
 
   def search_params
-    {
-      start: 0,
-      count: PAGE_SIZE_TO_GET_EVERYTHING,
-      filter_taxons: [content_id]
-    }
+    if document_type.nil?
```

Comment:
There's some repetition here as the params are almost identical for each case apart from the `filter_format`. Perhaps this could be simplified, something like:

```ruby
params = {
  start: 0,
  count: PAGE_SIZE_TO_GET_EVERYTHING,
  filter_taxons: [content_id]
}

if document_type
  params.merge!({filter_format: [document_type]})
end

params
```

Reply:
Ah nice. That does seem a more succinct way of doing it. I will change it.